### PR TITLE
Drop Python 3.6

### PR DESCRIPTION
**Pull request by lafrech**
- Drop Python 3.6.
- Don't use OrdereDict anymore since dict is ordered.

Python 3.6 EOLs on December 23rd 2021: https://endoflife.date/python

### CODE REVIEW

**Review by sloria — APPROVED** [review]

**Comment by lafrech** [comment]
I'll probably wait until Xmas to merge this so that 3.6 is EOL already.

**Comment by sloria** [comment]
yep, sounds good

**Pull request merged**
